packages/artifact/src/internal/shared/config.ts

```diff
@@ -57,3 +57,7 @@ export function getConcurrency(): number {
   const concurrency = 16 * numCPUs
   return concurrency > 300 ? 300 : concurrency
 }
+
+export function getUploadChunkTimeout(): number {
+  return 30_000
```

Comment:
300000?

Reply:
If it's supposed to be from the earlier `const timeoutDuration = 300000 // 30 seconds`, could you add a similar comment as to what this value is supposed to be. Or `getUploadChunkTimeout` could be renamed to something like `getUploadChunkTimeoutInMilliseconds`

Reply:
The original chunk timeout was incorrectly set to 300000 ms, which is 300 seconds:
https://github.com/actions/toolkit/blob/1db73622df8a649a01cb12b8a217c903419009fa/packages/artifact/src/internal/upload/blob-upload.ts#L50

It was intended to be 30 seconds from the comment, which is why I used the numeric separator (`_`) to make it a bit more obvious

Reply:
Ah, nice catch! Could you then just add the same comment with `// 30 seconds` so anyone stumbling by can better understand it

Reply:
done! https://github.com/actions/toolkit/pull/1774/commits/3e34f6d19cd8feeb813dd7bfb0de5d75aaa43565